frontend/packages/integration-tests-cypress/views/common.ts

```diff
@@ -0,0 +1,2 @@
+export const resourceStatusShouldBe = (desiredState: string) =>
```

Comment:
Hi, really should be `resourceStatusShouldContain` as that is the funciton.  `...ShouldBe` implies 'equals'.